Cargo.toml

```
@@ -4,7 +4,8 @@ default-members = ["pkgs/cli"]
 members = [
     "pkgs/cli",
     "pkgs/merkle-tree-wasm",
-    "pkgs/spartan-ecdsa-wasm"
+    "pkgs/spartan-ecdsa-wasm", 
+    "pkgs/circuits/halo2-ecdsa"
```

hmm
not sure you should have chained this PR on your halo2 PR. It is totally unrelated. Please make these PRs separate.
```suggestion
    "pkgs/spartan-ecdsa-wasm"
```